tests/mitol/google_sheets/test_sheet_handler_api.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from pytest_lazyfixture import lazy_fixture
+
+from mitol.google_sheets.sheet_handler_api import SheetHandler
+
+
+@pytest.mark.usefixtures("google_sheets_base_settings")
+@pytest.mark.parametrize(
+    "_creds",
+    [
+        lazy_fixture("google_sheets_service_creds_settings"),
+        lazy_fixture("google_sheets_client_creds_settings"),
+    ],
+)
+def test_is_configured_missing(_creds):
```

Comment:
Does the underscore here indicate that the parameter is not used by the code within the test?

Reply:
Yeah, it tells linters not to worry about it, `pytest` requires the parameter to be present on the signature, but there's no need for the value, just for the side effects of the fixture.